internal/core/src/segcore/load_index_c.cpp

```diff
@@ -25,6 +25,11 @@
 #include "storage/RemoteChunkManagerSingleton.h"
 #include "storage/LocalChunkManagerSingleton.h"
 
+bool
+IsLoadWithDisk(const char* index_type, int index_engine_version) {
```

Comment:
Is it possible nullptr for index_type here? Knowhere function will convert index_type into string

Reply:
the `index_type` will be freed after the function call, but `std::string` constructor will copy the data `index_type` points, so it would work fine